manim/mobject/svg/text_mobject.py

```diff
@@ -1186,6 +1222,10 @@ def __init__(
             text_without_tabs = text.replace("\t", " " * self.tab_width)
 
         colormap = self.extract_color_tags()
+        if len(colormap) > 0:
+            logging.getLogger("manim").warning(
```

Comment:
import logger from root don't do this, possibly logger should already be defined.
```suggestion
            logger.warning(
```

Reply:
Good catch. I will do that with a separate commit, thus removing the unnecessary import.